--- local.mak	2018-03-31 16:20:56.223725517 +1030
+++ local.mak	2018-03-31 16:19:58.296565265 +1030
@@ -1,2 +1,14 @@
 MONO_SIGNCODE = echo ***Bypassing signcode***
 GPG_SIGN = echo ***Bypassing gpg***
+
+SHELL := /bin/bash
+CC := /usr/bin/i686-w64-mingw32-gcc
+GMSGFMT := msgfmt
+MAKENSIS := /usr/bin/makensis
+WINDRES := /usr/bin/i686-w64-mingw32-windres
+STRIP := /usr/bin/i686-w64-mingw32-strip
+INTLTOOL_MERGE := /usr/bin/intltool-merge
+LDFLAGS := -static-libgcc
+
+INCLUDE_PATHS := -I$(PIDGIN_TREE_TOP)/../win32-dev/w32api/include
+LIB_PATHS := -L$(PIDGIN_TREE_TOP)/../win32-dev/w32api/lib
--- libpurple/win32/global.mak	2016-06-19 09:41:02.000000000 +0930
+++ libpurple/win32/global.mak	2018-03-31 15:39:10.261537954 +1030
@@ -20,7 +20,7 @@
 PERL_LIB_TOP ?= $(WIN32_DEV_TOP)/perl-5.20.1.1
 SILC_TOOLKIT ?= $(WIN32_DEV_TOP)/silc-toolkit-1.1.12
 GSTREAMER_TOP ?= $(WIN32_DEV_TOP)/gstreamer-0.10.13
-GCC_SSP_TOP ?= $(shell dirname $(shell which $(CC)))
+GCC_SSP_TOP ?= $(shell $(CC) -print-search-dirs | grep install | cut -d " " -f 2-)
 CYRUS_SASL_TOP ?= $(WIN32_DEV_TOP)/cyrus-sasl-2.1.26_daa1
 
 # Where we installing this stuff to?
